extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -95,4 +97,21 @@ require([
 	} else {
 		setupPlayer();
 	}
+
+	// XW-4157 PageFair causes pausing the video, as a workaround we play video again when it's paused
+	window.addEventListener('wikia.blocking', function () {
+		if(playerInstance) {
+			if(playerInstance.getState() === 'paused') {
+				playerInstance.play();
```

Comment:
Please make sure non-autoplay still works after these changes.

Reply:
it still works fine